### ISSUE TITLE
The scroll function don't work well with shapes in the latest version

### ISSUE THREAD

**Issue by diegovdc** [ojack/hydra-synth#62]
For example this: `shape(3).scrollX(0.1, 0.1).out()`. I would expect the triangle to come out on the other side, but it doesn't.
I am using the `unpkg` version of the lib, because I want to dynamically load hydra, but I've also checked this code in https://hydra.ojack.xyz/, with similar results:

```html
<!DOCTYPE html>
<html lang="en">
<script src="https://unpkg.com/hydra-synth@1.3.0/dist/hydra-synth.js"></script>
<body>
  <canvas id="hydra-canvas"></canvas>
  <script>
    new Hydra({ canvas: document.getElementById('hydra-canvas') });
    shape(3).scrollX(0.1, 0.1).out();
  </script>
</body>
</html>
```

**Comment by micuat** [comment]
fixed in the current version (at least on 1.3.4)

**Comment by diegovdc** [comment]
Great, thanks
